docs/docsource/operation/GUI/Settings.rst

```diff
@@ -0,0 +1,20 @@
+.. _settings:
+
+Settings
+========
+
+The Settings window provides information on the |product| instance (its name, ID, 
```

Comment:
This is just out of curiosity, but do you know why the references to `|product|` are blue here and not in any other file?

Reply:
Your eye is keen! The other files are .txt, and thus without rules. Here, it's in an actual .rst file, so it's parsed as an rst variable/substitution.